src/core_plugins/kibana/public/dashboard/index.js

```diff
@@ -18,6 +18,7 @@ import uiModules from 'ui/modules';
 import indexTemplate from 'plugins/kibana/dashboard/index.html';
 import { savedDashboardRegister } from 'plugins/kibana/dashboard/services/saved_dashboard_register';
 import { createPanelState } from 'plugins/kibana/dashboard/components/panel/lib/panel_state';
+import { DashboardConsts } from './dashboard_consts';
```

Comment:
Can we name the file and variable without abbreviating "constants"? e.g. `DashboardConstants` and `./dashboard_constants`. There's not much gained by abbreviating them and "const" is a keyword thatI think is worth differentiating from the concept of "constants".